packages/bsky/src/api/app/bsky/graph/getSuggestedFollowsByActor.ts

```diff
@@ -46,13 +62,32 @@ const skeleton = async (input: SkeletonFnInput<Context, Params>) => {
   if (!relativeToDid) {
     throw new InvalidRequestError('Actor not found')
   }
-  const { dids, cursor } = await ctx.hydrator.dataplane.getFollowSuggestions({
-    actorDid: params.hydrateCtx.viewer,
-    relativeToDid,
-  })
-  return {
-    suggestedDids: dids,
-    cursor: cursor || undefined,
+
+  const statsigUser = await didToStatsigUser(params.hydrateCtx.viewer)
+  if (
+    ctx.suggestionsAgent &&
+    Statsig.checkGateSync(statsigUser, gates.newSuggestedFollowsByActor)
```

Comment:
My understanding is that this would throw if statsig isn't configured, which is allowed by the appview configuration.  In prod, for example, there's currently a `suggestionsAgent` but no statsig config, so I believe this endpoint would be throwing 500s.  If we can fail open here instead that would be awesome.

Reply:
Ah true, should have a fallback 👍